packages/terra-image/src/Image.module.scss

```diff
@@ -1,3 +1,7 @@
+// Themes
```

Comment:
nitpick - have we been adding this comment for every theme context pr? i don't think it's necessary, the imports are pretty self explanatory